docs/introduction/getting_started.rst

```diff
@@ -27,22 +48,37 @@ First we will construct two :ref:`site_documentation` objects for the :math:`^1\
     from mrsimulator.spin_system.tensors import SymmetricTensor
 
     # Create the Site objects
-    H_site = Site(isotope="1H")
+    H_site = Site(isotope = "1H")
     C_site = Site(
-        isotope="13C",
-        isotropic_chemical_shift=100.0,  # in ppm
-        shielding_symmetric=SymmetricTensor(
-            zeta=70.0,  # in ppm
-            eta=0.5,
+        isotope = "13C",
+        isotropic_chemical_shift = 100.0,  # in ppm
```

Comment:
again there is the spacing betwen `=` question

Reply:
my preference is space around all equal signs, but i defer to Deepansh's preference thatvwe follow pep